sdk/src/lib.rs

```diff
@@ -30,8 +31,10 @@ pub mod env;
 pub mod store;
 
 /// Transaction Management Module
+#[cfg(any(feature = "std", feature = "wasm"))]
```

Comment:
This is the type of thing I think feels wrong. Why if it can work in WASM/no_std do we need to specify either of these? Just add the relevant packages `std` feature to our SDK's `std` feature?

Reply:
The `std` and `wasm` features have `dep:subxt`, which isn't no_std compatible.

Reply:
I believe parts of subxt are actually @Serial-ATA, can you double check. `subxt_core` should be: https://github.com/paritytech/subxt/issues/1479?

Reply:
Yeah that works for `tangle_subxt`. I've been trying to convert that to have an `std` feature. In this case though, the only thing that's in `tx` relies on `OnlineClient` which is only available in `subxt`.

Reply:
This doesn't yet compile but I'm imagining in `tangle-subxt` something like
```
[dependencies]
parity-scale-codec = { workspace = true, default-features = false }
scale-info = { workspace = true, default-features = false }
subxt = { version = "0.37.0", default-features = false, optional = true }
subxt-signer = { version = "0.37.0", default-features = false, optional = true }
serde = { workspace = true, features = ["derive"], optional = true }

[features]
default = ["std"]
std = [
    "serde",
    "subxt/native",
    "subxt-signer/std"
]
web = [
    "subxt/web",
    "subxt-signer/web"
]
```

Reply:
Yeah, that's pretty much what I have locally (still WIP). For the `tx` module though, there's nothing in there that can be used in `no_std` right now.